pkgs/by-name/sp/speki/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  libiconv,
+  glib,
+  gtk3,
+  libsoup_3,
+  webkitgtk_4_1,
+  xdotool,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "speki";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "tbs1996";
+    repo = "spekispace";
+    hash = "sha256-bv3WOw1FyfBTombSqyAVEWnP092s8OQzkKr5TwnyplE=";
+    rev = "cc11bce95f63eda31f4e5fdf9df9736a36fe17b5";
```

And a little stylistic convention: put the hash last
```suggestion
    rev = "cc11bce95f63eda31f4e5fdf9df9736a36fe17b5";
    hash = "sha256-bv3WOw1FyfBTombSqyAVEWnP092s8OQzkKr5TwnyplE=";
```